toast-tk-runtime/src/main/java/io/toast/tk/runtime/AbstractScenarioRunner.java

```diff
@@ -14,7 +22,9 @@
 
 import io.toast.tk.core.annotation.EngineEventBus;
 import io.toast.tk.core.rest.RestUtils;
+import io.toast.tk.dao.domain.impl.test.block.ICampaign;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.toast.tk.dao.domain.impl.test.block.ICampaign'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)